src/backend/common/util.hpp

```diff
@@ -15,3 +15,7 @@
 #pragma once
 
 std::string getEnvVar(const std::string &key);
+
+// Dump the kernel sources only if the environment variable is defined
+static const char* saveJitKernelsEnvVarName = "AF_DUMP_JIT_KERNELS_DIR";
```

Comment:
I prefer to let you officially publish that option in the doc when the specs will be totally stable.